src/test/java/org/jabref/logic/importer/fetcher/IsbnViaOttoBibFetcherTest.java

```diff
@@ -0,0 +1,82 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.util.Optional;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.ImportFormatPreferences;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BiblatexEntryTypes;
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Answers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+
+@FetcherTest
+public class IsbnViaOttoBibFetcherTest extends AbstractIsbnFetcherTest {
+
+    @BeforeEach
+    public void setUp() {
+        bibEntry = new BibEntry();
+        bibEntry.setType(BiblatexEntryTypes.BOOK);
+        bibEntry.setField("bibtexkey", "9782819502746");
+        bibEntry.setField("title", "Les mots du passé : roman");
+        bibEntry.setField("publisher", "́Éd. les Nouveaux auteurs");
+        bibEntry.setField("year", "2012");
+        bibEntry.setField("author", "Denis");
+        bibEntry.setField("isbn", "978-2-8195-02746");
+        bibEntry.setField("url", "https://www.ottobib.com/isbn/9782819502746/bibtex");
+
+        fetcher = new IsbnViaOttoBibFetcher(mock(ImportFormatPreferences.class, Answers.RETURNS_DEEP_STUBS));
```

Comment:
Move `fetcher` into the respective tests to adhere to the 'given-then-when' structure of tests.

Reply:
Just out of interest to learn something, but doesn't this give (needless ?) code duplication especially since all tests are of the form "for-input-x-verify-output-y". If the state of the fetcher needs to be varied then a test of the form
```java
    @Test
    public void test() throws FetcherException {
        fetcher.setSomething(value);

        Entry fetchedEntry = fetcher.fetch(input);

        assertEquals(expected, fetchedEntry);
    }
```
still perfectly adheres to the usual test pattern.